layouts/partials/scripts.html

```diff
@@ -73,6 +73,7 @@
 <!-- scripts for prism -->
 <script src='{{ "js/prism.js" | relURL }}'></script>
 {{ end }}
+<script type="module" src='{{ "js/yall.js" | relURL }}'></script>
```

Comment:
Is this a lazy loading library. You may be able to write a script that uses intersection observer instead.

Reply:
yes - using https://github.com/malchata/yall.js . Does it look like I am importing that correctly? I could probably write my own logic... yes... but why reinvent the wheel?